docs/source/raft_configuration.md

```diff
@@ -2,8 +2,6 @@
 
 **Audience**: *Raft ordering node admins*
 
-Note: this topic describes the process for configuring a Raft ordering service that has not been bootstrapped with a system channel genesis block. For a version of this topic that includes information about the system channel, check out [Configuring and operating a Raft ordering service](https://hyperledger-fabric.readthedocs.io/en/release-2.2/raft_configuration.html).
```

Comment:
add a comment about "system channel not supported", but keep the reference to legacy version.